Sources/Specs/GameSpec.swift

```diff
@@ -0,0 +1,18 @@
+import Dictionary
+import SnowManLogic
+import Swiftest
+
+class GameSpec: Swiftest.Spec {
+  	let spec = describe("game logic") {
+  		let dictionary = DictionaryManager()
+		let game = Game(dictionary: dictionary)
+	
+    	describe("#getWord") {
+    		it("should return a word from the dictionary") {
+    			let word = game.getWord()
+    			let gameWords = words
+    			expect(gameWords).to.contain(word)
```

Comment:
This test is really just testing the functionality of the dictionary manager, which is [already tested here](https://github.com/NicoleCarpenter/SnowMan/pull/8/files#diff-6cdc6306eacff51947b84f8f42709891R16)